### PR TITLE
Safari Mobile 12.2 supports <input inputmode=

### PR DESCRIPTION
**Pull request by MorrisJohns**
Although I haven't added (because I haven't tested):  download attribute, Web Share API, Datalist, Color input type (as mentioned in https://twitter.com/daytonlowell/status/1089165318646349825 )

I did test inputmode on iPhone and iPad, but I didn't test if it has any bugs.

### CODE REVIEW

File: html/global_attributes.json [hunk 1]
```diff
@@ -836,7 +836,10 @@
           }
         }
       },
-      "inputmode": {
+      "
+      
+      
+      ": {
```
Comment (vinyldarkscratch) [inline html/global_attributes.json]: Whoa, what happened along here?  If we can get these lines reverted back to normal, then I'd say this looks perfect!

Comment (MorrisJohns) [reply html/global_attributes.json]: Ummm - clearly I need a LOT more practice before making my first change! I do appreciate your friendly feedback. I'll close this, stand in the corner, put the dunce hat on, and make an appointment with my counsellor ;-) Cheers

**Comment by MorrisJohns** [comment]
Ummmm, clearly I need to practice my pull-requests some more...

**Comment by MorrisJohns** [comment]
Ignore this - I need to grok github change requests better!

**Comment by vinyldarkscratch** [comment]
Haha, you’re all gooed!  Everyone makes mistakes, and hey, you took the time to send your findings to this repository, letting others know about the data.  😉

**Pull request closed without merging**